pkgs/top-level/ocaml-packages.nix

```diff
@@ -1689,6 +1689,10 @@ in let inherit (pkgs) callPackage; in rec
 
   ocamlPackages_4_14 = mkOcamlPackages (callPackage ../development/compilers/ocaml/4.14.nix { });
 
+  ocamlPackages_4_14_unsafe_string = mkOcamlPackages (callPackage ../development/compilers/ocaml/4.14.nix {
+    unsafeStringSupport = true;
+  });
```

Comment:
Please add a comment why this is useful and/or necessary.

Reply:
I think I'll add comments in `pkgs/development/compilers/ocaml/generic.nix`, and I'm actually wondering if I should do the same for 4.13 (mldonkey) and 4.11 (weidu).

Reply:
If they're used similarly to the alias you added, by all means! Specifically named attrs like this make the code clearer IMO.

I think I'd prefer a comment next to the attrs themselves though, not some detached place with a similar topic.